Userland/Libraries/LibGfx/ImageFormats/WebPLoaderLossy.cpp

```diff
@@ -99,16 +101,312 @@ ErrorOr<VP8Header> decode_webp_chunk_VP8_header(ReadonlyBytes vp8_data)
     return VP8Header { version, show_frame, size_of_first_partition, width, horizontal_scale, height, vertical_scale, vp8_data.slice(10) };
 }
 
+namespace {
+
+// Reads n bits followed by a sign bit (0: positive, 1: negative).
+ErrorOr<i8> read_signed_literal(BooleanDecoder& decoder, u8 n)
```

Comment:
Does this method need help to ensure we don't truncate a value that's > 7 bits?

Reply:
Added a `VERIFY(n <= 7)` (the 5 callers this function currently has are all there ever will be; all 5 hardcode a fixed n value <= 7.)